charts/pvc-autoresizer/values.yaml

```diff
@@ -33,3 +33,9 @@ controller:
 
   # controller.terminationGracePeriodSeconds -- Specify terminationGracePeriodSeconds.
   terminationGracePeriodSeconds:  # 10
+
+# tolerations -- Ensure pods are not scheduled on inappropriate nodes
+tolerations: {}
+
+# nodeSelector -- Map of key-value pairs for scheduling pods on specific nodes
+nodeSelector: {}
```

Comment:
We currently have only one controller deployment, however, there would be more deployments in the future.
Therefore, these maps should be under the controller key, IMO

Reply:
Done!